src/iiif/tools.py

```diff
@@ -184,13 +194,18 @@ def img_is_public(metadata, document_barcode):
     raise DocumentNotFoundInMetadataError()
 
 
-def create_mail_login_token(email_address, key, expiry_hours=24):
+def create_mail_login_token(email_address, origin_url, key, expiry_hours=24):
     """
     For burgers to be able to login they can send their email address with which we create a
```

Comment:
burgers -> citizens

Reply:
This method description could use some rewording, as I do not understand it completely. Something along the lines of:
```
Prepare a JSON web token to be used by dataportaal. This enables citizens to login into ....
```